docs/src/pages/[platform]/connected-components/storage/fileuploader/examples/FileUploaderErrorExample.tsx

```diff
@@ -1,18 +1,23 @@
+import * as React from 'react';
 import { FileUploader } from '@aws-amplify/ui-react';
 
 export const FileUploaderErrorExample = () => {
-  const onError = (event) => {
-    console.log(event);
+  const [message, setMessage] = React.useState('');
+  const onError = (error) => {
+    setMessage(`${error}`);
```

Comment:
Changing this to show the message in the example so users don't need to open the console to see the effect.